src/NuGet.Core/NuGet.Commands/CommandArgs/PackArgs.cs

```diff
@@ -11,6 +11,9 @@
 
 namespace NuGet.Commands
 {
+    /// <summary>
+    /// Pack arguments shared between dotnet.exe and msbuild.exe /t:pack
```

Comment:
nuget.exe uses it too, so everything that can pack uses it. The name is `PackArgs`, so a comment saying "pack arguments" doesn't seem very helpful. Sorry, but I'm not a fan of obvious comments.

Reply:
I agree with Andy. 
The PackArgs are obviously used by the PackCommandRunner which is unique.

Reply:
Ok, I will remove it.